src/AdminDashboard/Devices/Sections/DangerZone.js

```diff
@@ -7,20 +7,29 @@ class DangerZone extends Component {
     wipe = async () => {
         const isOK = await Confirmation.isOK(this.wipeDevice)
         if (isOK) {
-            this.props.showNotification('Success', 'Data deleted successfully')
+            try {
+                const response = await this.props.glpi.updateItem({
+                    itemtype: 'PluginFlyvemdmAgent',
```

Comment:
Could you provide a JSON with all the item types specific of the plugin?
So you can change it in the future easily.

Reply:
Sure!